accepted/2024/vulnerabilities-in-packages.config-restore.md

```diff
@@ -0,0 +1,93 @@
+# ***Vulnerabilities in packages.config restore***
+
+- [Nikolche Kolev](https://github.com/nkolev92)
+- GitHub Issue <https://github.com/NuGet/Home/issues/12307>
+
+## Summary
+
+This is a continuation of the [vulnerabilities in PackageReference work](../2022/vulnerabilities-in-restore.md), extending matching functionality for packages.config restore.
+
+## Motivation
+
+See [vulnerabilities in PackageReference restore motivation](../2022/vulnerabilities-in-restore.md#motivation).
+
+## Explanation
+
+### Functional explanation
+
+Packages.config restore combines all packages regardless of project. Instead of raising warnings and errors on a per project level, at restore time, they're raised on a per package level.
+
+The configuration knobs for packages.config restore auditing functionality will be NuGet.config based.
+In particular, there will be 2 new configuration keys:
```

Comment:
I disagree with this.

Firstly, as you already listed in the unanswered questions section, should PackageReference projects use this config?

Secondly, from a customer point of view, it means they'll have to duplicate the config (when not using defaults), and will have (at least) two places to update when changing values. This will have even higher impact when [suppressing specific advisories](https://github.com/NuGet/Home/pull/12921) is implemented, as that feature will have to design a packages.config specific alternative, and customers may need to duplicate the suppressions.

packages.config already reads MSBuild properties for lock files, so configuring and using lock files is the same, regardless if the project uses packages.config or PackageReference.  I think NuGetAudit for packages.config should be the same.

Reply:
Usage of packages.config lock is minimal, so I'm not sure it's necessarily a well established pattern. 
Maybe we make the nuget.config apply to everything? 

I'm also wondering if the technical overhead of adding individual project support in packages.config is worth it in V1 of the feature.

Reply:
fwiw, I'll schedule a meeting to discuss this.

Reply:
is there any technical or feasibility issues with using the same pattern we've established earlier w/ MSBuild properties? 

I like re-using the same pattern if possible.

Reply:
My argument is that it's not a pattern we've adopted for packages.config projects. 
Technically, they're both feasible, but for packages.config projects using MSBuild property is not really a thing like it is in PackageReference.